pcdsdevices/epics/slits.py

```diff
@@ -44,13 +66,26 @@ class Slits(IocDevice):
     close_cmd = Component(EpicsSignal, ":CLOSE")
     block_cmd = Component(EpicsSignal, ":BLOCK")
 
+    #Nominal
+    nominal_aperature = 5.0
+
+    #Subscription information
+    SUB_AP_CH = 'aperature_changed'
+    _default_sub = SUB_AP_CH
+
     def __init__(self, prefix, *, ioc="", read_attrs=None, name=None,
                  **kwargs):
         if read_attrs is None:
-            read_attrs = ['xcenter', 'xwidth', 'ycenter', 'ywidth', 'blocked']
+            read_attrs = ['xcenter', 'xwidth', 'ycenter', 'ywidth']
         super().__init__(prefix, ioc=ioc, read_attrs=read_attrs, name=name,
                          **kwargs)
 
+        #Subscribe to changes in aperature
+        self.xwidth.readback.subscribe(self._aperature_changed,
+                                       run=False)
+        self.ywidth.readback.subscribe(self._aperature_changed,
+                                       run=False)
+
```

Comment:
We need to go back at some point and clean up all of our classes that subscribe during the init, when they don't really need to subscribe unless someone else subscribes to them. I have a feeling it's bad for pyepics to be unnecessarily monitoring things.

Reply:
Maybe correct. Not sure I want to worry about it until we actually see an issue.